### PR TITLE
Libraries and navbar

### PR DESCRIPTION
**Pull request by shaargtz**
Got bootstrap and fontawesome icons for a better looking page. Got the navbar going at the top.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,16 +4,33 @@
...
+          <a class="nav-item nav-link" href="#">Gallery</a>
+          <a class="nav-item nav-link" href="#">Fun</a>
+        </div>
+      </div>
+    </nav>
     <div id="content">
       <h1>Mariano Shaar</h1>
       <a href="https://github.com/shaargtz" class="link">
-        <img src="./images/icons/github.svg" alt="GitHub link icon" class="icon">
+        <i class="fab fa-github"></i>
```
Comment (gloriapark) [inline portfolio/src/main/webapp/index.html:30]: Can you still include the alt field for these icons as well? In general we want to include the alt fallback tag for all multimedia (images, videos, animations, icons). If the fontawesome icons already have the alt tag then that's fine then! (And same goes for the linkedIn icon below)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,16 +4,33 @@
...
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">Portfolio</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+        <div class="navbar-nav">
+          <a class="nav-item nav-link active" href="#">About me <span class="sr-only">(current)</span></a>
```
Comment (gloriapark) [inline portfolio/src/main/webapp/index.html:20]: nit: Remove trailing whitespace after "me"

Comment (shaargtz) [reply portfolio/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shaargtz) [reply portfolio/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,16 +4,33 @@
     <meta charset="UTF-8">
     <title>3030</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```
[the remark this answers is not in the document]
Comment (shaargtz) [reply portfolio/src/main/webapp/index.html]: Yep, it didn't have the rel="stylesheet" so I had to add it. Done!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -4,16 +4,33 @@
     <meta charset="UTF-8">
     <title>3030</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```
Comment (gloriapark) [inline portfolio/src/main/webapp/index.html:7]: nit: Oops it looks like rel=stylesheet is included twice now!

Comment (shaargtz) [reply portfolio/src/main/webapp/index.html]: Oh I scrolled past it so I hadn't seen it, my bad. Done!

**Pull request merged**